### DISCUSSION TITLE
Controller Becomes Unresponsive After Setting New Config Value For Battery Device

### DISCUSSION THREAD

**Discussion by ADHDSquir** [zwave-js/zwave-js#6951] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
When setting a new configuration value for a battery-powered device, the value is not set and the controller becomes unresponsive to any other commands for any other devices.

What did you expect to happen instead?
The network should continue to function while waiting for a battery powered device to wake up.

1. Select a battery powered device
2. Change a Configuration value and click the send arrow
3. The spinning circle says to "wake up your device in order to send commands"
4. Waking the device doesn't cause the commands to send
5. No command can be sent to any other device on the network until Z-Wave JS is restarted

Anything else we should know?

### Software versions

Driver (node-zwave-js): 12.11.1

Z-Wave JS UI: 9.14.1.7018936

### Device information

Manufacturer: Zooz
Model name: ZSE11 Q Sensor
Node ID: 58

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_current.log](https://github.com/user-attachments/files/15936369/zwavejs_current.log)

**Comment by AlCalzone** [comment, marked as answer]
Thanks for reporting, I can reproduce. Escalated to an issue: https://github.com/zwave-js/node-zwave-js/issues/6952
